src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { SlideMenu } from "../SlideMenu/SlideMenu";
```

Comment:
SlideMenu 컴포넌트를 default export 하셨네요. default export의 경우,  중간에 { }중괄호가 필요하지 않습니다 :) 
```suggestion
import SlideMenu from "../SlideMenu/SlideMenu";
```